apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -119,13 +129,73 @@ impl Default for MetricsCommon {
             service_name: None,
             service_namespace: None,
             resource: BTreeMap::new(),
+            views: Vec::with_capacity(0),
             buckets: vec![
                 0.001, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5, 1.0, 5.0, 10.0,
             ],
         }
     }
 }
 
+#[derive(Clone, Debug, Deserialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct MetricView {
+    /// The instrument name you're targeting
+    pub(crate) instrument_name: String,
```

Comment:
```suggestion
    pub(crate) name: String,
```

This is called `name` in the spec: https://opentelemetry.io/docs/specs/otel/metrics/sdk/#instrument-selection-criteria

In addition it should be defaulted to "*".

Reply:
Name updated, I think we should keep the `name` mandatory, if not then every fields would be optional and that could be weird. I'll document it to let people know they can use wildcard in names